Makefile

```diff
@@ -50,10 +59,17 @@ endif
 .PHONY: api-default
 api-default:
 ifeq ("$(wildcard $(GO_EXEC))", "")
-	@echo "ERROR: Need to install golang 1.13+ first"
+	@echo "ERROR: Need to install golang 1.16+ first"
 	exit 1
 endif
-
+	@if [ "${GO_MAJOR}" -lt "${REQUIRED_MIN_GO_MAJOR}" ] ; then \
+		echo "ERROR: Need to install golang 1.16+ first"; \
+		exit 1; \
+	fi
+	@if [ "${GO_MINOR}" -lt "${REQUIRED_MIN_GO_MINOR}" ] ; then \
+		echo "ERROR: Need to install golang 1.16+ first"; \
+		exit 1; \
+	fi
```

Comment:
The logic will make a mistake if we use the go version like 2.0 in the future.
: )

Reply:
Why I wrote such horrible logic🤣🤣
Changing ASAP

Reply:
Why I wrote such horrible logic🤣🤣
Changing ASAP

Reply:
fixed